changes.txt

```diff
@@ -24,7 +24,7 @@ Aug 29th, 2017
 v1.0.1
        -- Dmitry Kovalenko provided fixes to mp_add_d() and mp_init_copy()
        -- Matt Johnston contributed some improvements to mp_div_2d(),
-          mp_exptmod_fast(), mp_mod() and mp_mulmod()
+          s_mp_exptmod_fast(), mp_mod() and mp_mulmod()
```

Comment:
I'd say we should keep this file as is and only add to it on a new release

Reply:
Ok!